### PR TITLE
AIShield AI Security Monitoring Solution updates to version 3.1.0

### PR DESCRIPTION
**Pull request by udanashivm**
Required items, please complete
   Updating 'AIShield AI Security Monitoring' Solution to version 3.1.0
  
   Change(s):
 -  Adding and updating Analytic Rules
 - Updated parser
 
   Reason for Change(s):
   -  Submitting updated content to the repository.

   Version Updated:
   -Yes. Updating to version 3.1.0. Added 4 Analytic Rules

   Testing Completed:
   -yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @udanashivm, Thanks for raising this PR. This PR will be investigated and will update you about the same before 28 December, 2023.

File: Solutions/AIShield AI Security Monitoring/Parsers/AIShield.yaml [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
         AttackName = attack_name_s,
         Message = RawMessage_s,
         ServiceName = service_name_s,
         AssetId = asset_id_s,
         EventTimestamp = timestamp_t,
-        SourceName = source_name_s
+        SourceName = source_name_s,
+        SuspiciousLevel = suspicious_level_s,
+        Probability = probability_d
     | project-away probability_d
```
Comment (v-prasadboke) [inline Solutions/AIShield AI Security Monitoring/Parsers/AIShield.yaml:23]: Hello @udanashivm, project-away should be column Probability. as column probability_d is renamed to Probability

Comment (v-prasadboke) [reply Solutions/AIShield AI Security Monitoring/Parsers/AIShield.yaml:23]: I'll commit this change, probability_d is renamed to Probability. As this is causing parser failure.

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

File: Solutions/AIShield AI Security Monitoring/Parsers/AIShield.yaml (hunk 1 above)
Comment (udanashivm) [reply Solutions/AIShield AI Security Monitoring/Parsers/AIShield.yaml]: Thank you for the approval & merge @v-prasadboke  :)

**Pull request merged**